### PR TITLE
Fix account creation page

### PR DESCRIPTION
**Pull request by koivyr**
### Summary
- #125 
- #126 

### Test Plan
-

### CODE REVIEW

File: donut/modules/account/routes.py [hunk 1]
```diff
@@ -60,18 +60,16 @@ def create_account_submit(create_account_key):
...
+        or password2 is None:
         flask.current_app.logger.warn(
             f'Invalid create account form for user ID {user_id}')
         flask.flash("Invalid request.")
         return flask.redirect(flask.url_for("home"))
 
-    if helpers.handle_create_account(user_id, username, password, password2,
-                                     birthday):
+    if helpers.handle_create_account(user_id, username, password, password2):
+        flask.session['username'] = username
```
Comment (calebsander) [inline donut/modules/account/routes.py:72]: We should probably also automatically log people in on the reset password page

File: donut/modules/account/templates/create_account.html [hunk 2]
```diff
@@ -1,52 +1,53 @@
...
+          <div class="form-group">
+            <label for="password" class="required">Password</label>
+            <input type="password" name="password" class="form-control" />
+          </div>
+          <div>
+            <label class="required">Confirm password</label>
+            <input type="password" name="password2" class="form-control" />
+         </div>
+       </div>
+       <p>Please verify your personal information. Email team@hacktech.io for issues</p>
```
Comment (calebsander) [inline donut/modules/account/templates/create_account.html]: lolol

File: donut/modules/account/templates/create_account.html [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
+          <div class="form-group">
+		    <label for="username" class="required">Username</label>           
+            <input type="text" name="username" class="form-control"/>
+          </div>
+          <div class="form-group">
+            <label for="password" class="required">Password</label>
+            <input type="password" name="password" class="form-control" />
+          </div>
+          <div>
+            <label class="required">Confirm password</label>
```
Comment (calebsander) [inline donut/modules/account/templates/create_account.html]: `for="password2"` so clicking on it highlights the input

File: donut/modules/account/templates/create_account.html [hunk 4]
```diff
@@ -1,52 +1,53 @@
...
+            <input type="password" name="password2" class="form-control" />
+         </div>
+       </div>
+       <p>Please verify your personal information. Email team@hacktech.io for issues</p>
+       <div class="form">
+         <div class="form-group">
+           <label>First Name: </label>
+           <label>{{ user_data['first_name'] }}</label>
```
Comment (calebsander) [inline donut/modules/account/templates/create_account.html]: Remove the data values from `label` fields? That way it's clearer what's a label and what's a value

**Review by calebsander — CHANGES_REQUESTED** [review]

File: donut/modules/account/templates/create_account.html (hunk 2 above)
Comment (koivyr) [reply donut/modules/account/templates/create_account.html]: F my life

**Review by calebsander — DISMISSED** [review]

**Pull request merged**